docs/guide/dev/integration/vue/README.md

```diff
@@ -12,6 +12,335 @@ For licensing, see LICENSE.md.
 
 # CKEditor 4 WYSIWYG Editor Vue Integration
 
-The CKEditor 4 Vue Integration is currently under development. Progress of our work can be tracked in the [CKEditor4-Vue repository](https://github.com/ckeditor/ckeditor4-vue).
+<info-box info="">
+	This feature is provided through the <a href="https://www.npmjs.com/package/ckeditor4-vue"><code>ckeditor4-vue</code> npm package</a>.
+</info-box>
 
-We are open for any valuable feedback. Don't hesitate to post any ideas or feature requests on the [Vue Integration issue page](https://github.com/ckeditor/ckeditor4/issues/2861).
+CKEditor 4 offers a native Vue integration through the CKEditor 4 Vue component. It provides deep integration of CKEditor 4 and Vue that lets you use the native features of the WYSIWYG editor inside a Vue component. The CKEditor 4 Vue component is compatible with Vue.js 2.x.
+
+## Basic Usage
+
+In order to create an editor instance in Vue, install the `ckeditor4-vue` npm package as a dependency of your project:
+
+```plaintext
+npm install ckeditor4-vue
+```
+
+Now you need to enable the CKEditor Vue component in your application. There are 2 ways to do so:
+
+* [via a direct script include](#direct-script-include),
+* [by using ES6 module imports](#using-es6-modules).
+
+Optionally, you can [use the component locally](#using-component-locally).
```

Comment:
Using locally is not an alternative to direct script include or module imports but to `Vue.use` so I'm not sure if it should be listed here. Mentioning it in both sections as already done (with `info-box`) seems to be enough IMHO.